src/modules/UI/components/OptionIcon/OptionIcon.ui.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+import React, {Component} from 'react'
+import FAIcon from 'react-native-vector-icons/FontAwesome'
+import THEME from '../../../../theme/variables/airbitz.js'
+
+type Props = {
+  iconName: string
+}
+
+type State = {}
+
+export default class OptionIcon extends Component<Props, State> {
```

Comment:
I'm not entirely certain this will work
Not all icons come from the same sub-library, i.e. `'react-native-vector-icons/FontAwesome'`

Reply:
All the icons currently used for the wallet options are from FontAwesome so it does work for now.
If in the future we will start using other fonts for those icons, we can expand it.